### PR TITLE
{tools}[foss/2019a] Spyder v4.1.5 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by zarybnicky**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by zarybnicky** [comment]
Oops, checksums, sorry.

File: easybuild/easyconfigs/s/Spyder/Spyder-4.1.5-foss-2019a-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+    ('sphinxcontrib-qthelp', '1.0.3', {
+        'modulename': 'sphinxcontrib.qthelp',
+        'checksums': ['4c33767ee058b70dba89a6fc5c1892c0d57a54be67ddd3e7875a18d14cba5a72'],
+    }),
+    ('sphinxcontrib-serializinghtml', '1.1.4', {
+        'modulename': 'sphinxcontrib.serializinghtml',
+        'checksums': ['eaa0eccc86e982a9b939b2b82d12cc5d013385ba5eadcc7e4fed23f4405f77bc'],
+    }),
+    ('spyder', version, {
+        'preinstallopts': """sed -i "s/'pyqt5<5.13;python_version>=\"3\"',//g" setup.py && sed -i "s/'pyqtwebengine<5.13;python_version>=\"3\"',//g" setup.py && sed -i 's/jedi==0.17.1/jedi==0.17.2/g' setup.py && sed -i "s/JEDI_REQVER = '=0.17.1'/JEDI_REQVER = '=0.17.2'/g" spyder/dependencies.py && """,
```
Comment (boegel) [inline easybuild/easyconfigs/s/Spyder/Spyder-4.1.5-foss-2019a-Python-3.7.2.eb]: @zarybnicky Lines must fit in 120-char columns or the style check fails, so how about something like this (untested):

```suggestion
        'preinstallopts': ' && '.join([
            """sed -i "s/'pyqt5<5.13;python_version>=\"3\"',//g" setup.py""",
            """sed -i "s/'pyqtwebengine<5.13;python_version>=\"3\"',//g" setup.py""",
            """sed -i 's/jedi==0.17.1/jedi==0.17.2/g' setup.py""",
            """sed -i "s/JEDI_REQVER = '=0.17.1'/JEDI_REQVER = '=0.17.2'/g" spyder/dependencies.py""",
        ]),
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 15 out of 15 (1 easyconfigs in this PR)
node3125.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/191e4dc517db156bf6cefa5d25d2ed07 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in this PR)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/c8f8652e6b7e945fe7713bc07660c648 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in this PR)
node125.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/6d6970fd569d646cf4df30460b1ad0e0 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @zarybnicky!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2637.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/71f48be510df2b64f25b57abfbc251a9 for a full test report.

**Pull request merged**